bigtable/bigtable.go

```diff
@@ -295,13 +317,22 @@ func (ti *tableImpl) ApplyReadModifyWrite(ctx context.Context, row string, m *Re
 // By default, the yielded rows will contain all values in all cells.
 // Use RowFilter to limit the cells returned.
 func (t *Table) ReadRows(ctx context.Context, arg RowSet, f func(Row) bool, opts ...ReadOption) (err error) {
+	method := "cloud.google.com/go/bigtable.ReadRows"
```

Comment:
The method names should match existing method names:
- ReadRows
- SampleRowKeys
- MutateRow
- MutateRows
- CheckAndMutateRow
- etc

Reply:
Updated